contents/data/person.yaml

```diff
@@ -9,8 +9,18 @@
   twitter:
   website:
 
-- name: Mark Algee-Hewitt
```

Comment:
@VickyGarnett please be extra careful when dealing with records in person.yaml. Here you deleted Mark Algee and replaced him with Marc Aberle, which breaks Mark Algee's resource & prevents the site to be built. 

You don't have to fix this particular mistake, I will do it in a sec, but please keep this in mind.